### ISSUE TITLE
org-roam backlink is getting printed twice

### ISSUE THREAD

**Issue by anildigital** [org-roam/org-roam#1005]
### Description

org-roam backlink prints link twice

#### Steps to Reproduce

```
- Emacs: GNU Emacs 27.0.91 (build 1, x86_64-apple-darwin19.6.0, NS appkit-1894.60 Version 10.15.6 (Build 19G73))
 of 2020-07-29
- Framework: N/A
- Org: Org mode version 9.3 (release_9.3 @ /usr/local/Cellar/emacs-plus@27/27.0.91/share/emacs/27.0.91/lisp/org/)
- Org-roam: 1.2.1
```

<img width="987" alt="Screenshot 2020-08-06 at 8 04 57 AM" src="https://user-images.githubusercontent.com/266/89484858-10457680-d7bd-11ea-8751-f4d631ff228d.png">

**Comment by jethrokuan** [comment]
The first line is the outline hierarchy, and the second line is the preview content (which happens to be the same atm).

This is something we are hoping to change in the upcoming buffer revamp.

**Comment by anildigital** [comment]
Thanks. It looks weird though.

<img width="433" alt="image" src="https://user-images.githubusercontent.com/266/89724461-cb217e80-da20-11ea-8752-b0ed4b9633b8.png">

**Comment by jethrokuan** [comment]
Closed with #1655
